specification/sbi_cove.adoc

```diff
@@ -1816,113 +1816,113 @@ This interface specification is TBD.
+| <<sbi_covi_set_tvm_aia_cpu_imsic_addr, sbi_covi_set_tvm_aia_cpu_imsic_addr>> | 
 Set TVM CPU AIA address
 
-| <<sbi_tee_interrupt_convert_tvm_aia_imsic, sbi_tee_interrupt_convert_tvm_aia_imsic>> | Convert 
+| <<sbi_covi_convert_tvm_aia_imsic, sbi_covi_convert_tvm_aia_imsic>> | Convert 
 TVM GPA AIA address to confidential
 
-| <<sbi_tee_interrupt_reclaim_tvm_aia_imsic, sbi_tee_interrupt_reclaim_tvm_aia_imsic>> | 
+| <<sbi_covi_reclaim_tvm_aia_imsic, sbi_covi_reclaim_tvm_aia_imsic>> | 
 Reclaim TVM GPA AIA address from confidential
 
-| <<sbi_tee_interrupt_bind_aia_imsic, sbi_tee_interrupt_bind_aia_imsic>> |
+| <<sbi_covi_bind_aia_imsic, sbi_covi_bind_aia_imsic>> |
 Binds a TVM vCPU to the current physical CPU using the confidential guest interrupt file.
 
-| <<sbi_tee_interrupt_unbind_aia_imsic_begin, sbi_tee_interrupt_unbind_aia_imsic_begin>> |
+| <<sbi_covi_unbind_aia_imsic_begin, sbi_covi_unbind_aia_imsic_begin>> |
 Begins the unbind process for the specified vCPU from its guest interrupt file.
 
-| <<sbi_tee_interrupt_unbind_aia_imsic_end, sbi_tee_interrupt_unbind_aia_imsic_end>> |
+| <<sbi_covi_unbind_aia_imsic_end, sbi_covi_unbind_aia_imsic_end>> |
 Completes the unbind process for the specified vCPU from its guest interrupt files after
 a TLB flush sequence for the TVM has been completed.
 
-| <<sbi_tee_interrupt_inject_tvm_cpu, sbi_tee_interrupt_inject_tvm_cpu>> |
+| <<sbi_covi_inject_tvm_cpu, sbi_covi_inject_tvm_cpu>> |
 Injects an external interrupt with the given interrupt_id into the specified vCPU.
 
-| <<sbi_tee_interrupt_rebind_aia_imsic_begin, sbi_tee_interrupt_rebind_aia_imsic_begin>> |
+| <<sbi_covi_rebind_aia_imsic_begin, sbi_covi_rebind_aia_imsic_begin>> |
 Begins the rebinding process for the specified vCPU to the current physical CPU and the specified
 confidential guest interrupt file. The host must complete a TLB invalidation sequence
-for the TVM before cloning old interrupt file state using `sbi_tee_interrupt_rebind_aia_imsic_clone()`.
+for the TVM before cloning old interrupt file state using `sbi_covi_rebind_aia_imsic_clone()`.
 
-| <<sbi_tee_interrupt_rebind_aia_imsic_clone, sbi_tee_interrupt_rebind_aia_imsic_clone>> |
+| <<sbi_covi_rebind_aia_imsic_clone, sbi_covi_rebind_aia_imsic_clone>> |
 Clones the old guest interrupt file of the specified vCPU. Caller must make sure to invoke this from
 old physical CPU. The guest interrupt file after this is free to be reclaimed or bound to another
 vCPU.
 
-| <<sbi_tee_interrupt_rebind_aia_imsic_end, sbi_tee_interrupt_rebind_aia_imsic_end>> |
+| <<sbi_covi_rebind_aia_imsic_end, sbi_covi_rebind_aia_imsic_end>> |
 Completes the rebind process for the specified vCPU from this physical CPU and its guest
 interrupt files. Must be called from the same physical CPU as
-`sbi_tee_interrupt_rebind_aia_imsic_begin()`.
+`sbi_covi_rebind_aia_imsic_begin()`.
 
 |===
 
-=== Summary of TEEI - TG-ABI
+=== Summary of COVE Guest Extension (COVG) 
```

Comment:
```suggestion
=== Summary of CoVE Guest Extension (COVG) 
```

Reply:
Do you prefer to remove all references to T[GH]-ABI ? There are bunch of them including few in the figures. 
Same goes for TEEI - TEE Interface - Should we get rid of that acronym as well ?

Reply:
Done